tests/functional/test_group.py

```diff
@@ -0,0 +1,17 @@
+def test_group_list(run_line, load_api_fixtures, go_ep1_id):
+    """
+    Runs globus group list and validates results
+    """
+    data = load_api_fixtures("groups.yaml")
+
+    group1_id = data["metadata"]["group1_id"]
+    group2_id = data["metadata"]["group2_id"]
+    group1_name = data["metadata"]["group1_name"]
+    group2_name = data["metadata"]["group2_name"]
+
+    result = run_line("globus group list")
+
+    assert group1_id in result.output
```

Comment:
I wonder... Would it be useful to have a regex matching helper for tests like this?
Something like
```python
# some "matcher fixture"
assert matcher(
    r"^Group ID:\s+(.*)$", # a regex with capturing groups
    result.output, # input string
    groups=[group1_id]  # values compared against the captured groups, in order
)
```

Reply:
Hmmm, yeah that might be a good idea. I mostly looked at the output from running the command to make sure it looked good, but if something changed and messed it up this test likely wouldn't catch it

Reply:
I'll queue it up for later. It should be pretty easy to write something.